src/functions-reference/real-valued_basic_functions.qmd

```diff
@@ -1628,11 +1628,19 @@ proportion theta, defined by \begin{eqnarray*}
 \lambda_1, \ \log(1 - \theta) + \lambda_2\right). \end{eqnarray*}
 {{< since 2.6 >}}
 
-<!-- R; log_mix; (T1 theta, T2 lp1, T3 lp2); -->
-\index{{\tt \bfseries log\_mix }!{\tt (T1 theta, T2 lp1, T3 lp2): real}|hyperpage}
+<!-- R; log_mix; (T1 thetas, T2 lps); -->
+\index{{\tt \bfseries log\_mix }!{\tt (T1 thetas, T2 lps): real}|hyperpage}
+
+`R` **`log_mix`**`(T1 thetas, T2 lps)`<br>\newline
+
+Extension of the two-mixture case above to more densities.
+Return the log mixture of the log densities stored in `lps`,
+where each log density has the mixing proportion given by
+the same index in `thetas`.
```

Comment:
1.  `thetas` is presumably required to be a simplex, so that should be mentioned.
2. `thetas` and `lps` have to be the same length, so that should be mentioned.
3.  The result should be given explicitly in math, e.g., `log(SUM_n thetas[n] * exp(lps[n]))` or the more efficient but also more opaque `log_sum_exp(log(thetas) + lps)` (it'd be OK to include both if you think just one would be confusing).

Reply:
Oddly enough, it doesn't seem that the math library is checking that thetas is a simplex, just that it is bounded between 0 and 1. Possibly worthy of a math issue?

Reply:
> Extension of the two-mixture case above to more densities. Return the log mixture of the log densities stored in `lps`, where each log density has the mixing proportion given by the same index in `thetas`.

I would prefer this to say something about how this is intended to be looped across each density observation but is generalized to more than two densities. This connects the warning in the mixture modeling section about why we need to loop.